ipcs_domain/src/main/java/com/ipcs/service/SecurityService.java

```diff
@@ -17,6 +17,8 @@
  */
 public interface SecurityService {
 
+    public long authenticateLoginInfoWithID(String userName, String password);

```

Comment:
method authenticateLoginInfoWithID  name is misleading.
Please change name, like getAuthenticatedUserID